spec/decorators/sets_attributes_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe SetsAttributes do
+  subject { SetsAttributes.new(term) }
+  let(:term) { term_mock }
+  let(:test_param) {{
+    "label" => ["blah"]
+  }}
+  before do
+    stub_repository
+    allow(term).to receive(:attributes=)
+    allow(term).to receive(:attributes).and_return(test_param)
+    allow(term).to receive(:valid?).and_return(true)
+  end
+
+  describe "#set_languages" do
```

Comment:
Can you add a test to make sure the language is actually set on a label?  Probably use .language or .has_language if you can get an RDF::Literal in response in the test.